### ISSUE TITLE
How to performing Finetune on a network which has batch normalization layer?

### ISSUE THREAD

**Issue by wjf199105** [BVLC/caffe#4888]

**Comment by kli-nlpr** [comment]
same question
